packages/front-end/components/DataSources/DataSourceLogo.tsx

```diff
@@ -0,0 +1,245 @@
+import { SDKLanguage } from "back-end/types/sdk-connection";
```

Comment:
This file is just a copy pasta of the `SDKLanguageLogo` component right now, but I'll remove the unnecessary code when I add in all the logos